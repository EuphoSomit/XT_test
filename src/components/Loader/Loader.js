import React from 'react';

export const Loader = () => (
    <>
        <div className="overlay"></div>
        <div className="spinner-border text-light" role="status">
            <span className="sr-only">Loading...</span>
    </div>
  </>
)