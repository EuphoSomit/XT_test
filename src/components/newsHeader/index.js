import {NewsHeaderMemo as NewsHeader} from './NewsHeader';
export {NewsHeader as default}