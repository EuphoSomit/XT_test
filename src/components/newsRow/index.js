import {NewsRow} from './NewsRow';
export {NewsRow as default}