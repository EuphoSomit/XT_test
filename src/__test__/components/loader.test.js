import React from 'react';
import renderer from 'react-test-renderer';
import {Loader} from './../../components/Loader';

test('Loader component snapshot testing', () => {
    const component = renderer.create(
        <Loader />,
      );
    
    let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
   

})


