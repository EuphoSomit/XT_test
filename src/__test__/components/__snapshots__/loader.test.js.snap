// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component snapshot testing 1`] = `
Array [
  <div
    className="overlay"
  />,
  <div
    className="spinner-border text-light"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>,
]
`;
